test/e2e/memcached_test.go

```diff
@@ -82,25 +83,39 @@ func TestMemcached(t *testing.T) {
 		t.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
 	}
 
+	// get global framework variables
+	f := framework.Global
+	local := *f.ImageName == ""
+	if local {
+		*f.ImageName = "quay.io/example/memcached-operator:v0.0.1"
+	}
 	t.Log("Building operator docker image")
 	cmdOut, err = exec.Command("operator-sdk",
 		"build",
-		"quay.io/example/memcached-operator:v0.0.1").CombinedOutput()
+		*f.ImageName).CombinedOutput()
 	if err != nil {
 		t.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
 	}
-	operatorYAML, err := ioutil.ReadFile("deploy/operator.yaml")
-	if err != nil {
-		t.Fatal(err)
-	}
-	operatorYAML = bytes.Replace(operatorYAML, []byte("imagePullPolicy: Always"), []byte("imagePullPolicy: Never"), 1)
-	err = ioutil.WriteFile("deploy/operator.yaml", operatorYAML, os.FileMode(filemode))
-	if err != nil {
-		t.Fatal(err)
+	if local {
+		operatorYAML, err := ioutil.ReadFile("deploy/operator.yaml")
+		if err != nil {
+			t.Fatal(err)
+		}
+		operatorYAML = bytes.Replace(operatorYAML, []byte("imagePullPolicy: Always"), []byte("imagePullPolicy: Never"), 1)
+		err = ioutil.WriteFile("deploy/operator.yaml", operatorYAML, os.FileMode(filemode))
+		if err != nil {
+			t.Fatal(err)
+		}
+	} else {
+		t.Log("Pushing docker image to repo")
+		cmdOut, err = exec.Command("docker",
+			"push",
+			*f.ImageName).CombinedOutput()
```

Comment:
One small nit. Can we keep this in a single line. I don't think Go recommends any particular line length to conform to and this is well under a 100 chars. We have other one line statements that are longer than this.
So let's keep this and other statements to a single line:
```Go
cmdOut, err = exec.Command("docker", "push", *f.ImageName).CombinedOutput()
```